src/clj/rems/api/blacklist.clj

```diff
@@ -54,13 +60,14 @@
       :return schema/SuccessResponse
       (let [userid (user-mappings/find-userid (getx-in command [:blacklist/user :userid]))
             command (assoc-in command [:blacklist/user :userid] userid)]
-        (blacklist/add-user-to-blacklist! (getx-user-id) command)
-        (doseq [cmd (rejecter-bot/reject-all-applications-by userid)]
-          (let [result (command/command! cmd)]
-            (when (:errors result)
-              (log/error "Failure when running rejecter bot commands:" {:cmd cmd :result result})))))
-
-      (ok {:success true}))
+        (or (user-or-resource-not-found-error command)
```

Comment:
I think this would read slightly better, and like the other places where use `or`-pattern, if the error would be split into `invalid-user-error` and `invalid-resource-error`. (or `not-found-*-error`). No need to cram them together into one function?